airbyte-integrations/connectors/source-orb/source_orb/spec.json

```diff
@@ -0,0 +1,19 @@
+{
+  "documentationUrl": "https://docs.withorb.com/",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Orb Spec",
+    "type": "object",
+    "required": ["api_key"],
+    "additionalProperties": false,
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "title": "Orb API Key",
+        "description": "Orb API Key, issued from the Orb admin console.",
+        "airbyte_secret": true,
+        "order": 1
+      }
+    }
```

Comment:
Should there be an option to specify a "minimum" date to start incremental sync from if someone doesn't wish to get all data from the beginning?

Reply:
Yes, that sounds reasonable; I can add that as a parameter to the connector to avoid syncing from the beginning.